user_manual/files/webgui/sharing.rst

```diff
@@ -154,6 +154,9 @@ change     Allows uploading a new version of a shared file and replacing it
 delete     Allows the users you share with to delete shared files
 ========== ===================================================================
    
+
+.. _create_upload_only_link_share_label:
```

Comment:
I'll change the terminology to "Drop folder". However, I'll do it in a separate PR.